std/async/pool.ts

```diff
@@ -4,8 +4,13 @@
  * pooledMap transforms values from an (async) iterable into another async
  * iterable. The transforms are done concurrently, with a max concurrency
  * defined by the poolLimit.
- * 
- * @param poolLimit The maximum count of items being processed concurrently. 
+ *
+ * If an error is thrown from `iterableFn`, no new transformations will begin.
+ * All currently executing transformations are allowed to finish and still
+ * yielded on success. After that, the rejections among them are gathered and
+ * thrown by the iterator in an `AggregateError`.
```

Comment:
On one hand we want resource sanitation, so we let in progress jobs finish. On the other hand we want to fail fast like JS built-ins do, so we stop starting new jobs on error instead of trying all items and reporting all errors..

Reply:
SGTM